cli/e2e.js

```diff
@@ -206,9 +207,17 @@ function spawnBuild(argv, skyPagesConfig, webpack) {
  * @name e2e
  */
 function e2e(argv, skyPagesConfig, webpack) {
+  const specsPath = path.resolve(process.cwd(), 'e2e/**/*.spec.ts */');
+  const specsGlob = glob.sync(specsPath);
```

Comment:
I know I had them in this order in my example, but looking at it now, I think the two `const` definitions would be better after creating start and adding the listener.  So...

```
function e2e(argv, skyPagesConfig, webpack) {
  start = new Date().getTime();
  process.on('SIGINT', killServers);

  const specsPath = path.resolve(process.cwd(), 'e2e/**/*.e2e-spec.ts');
  const specsGlob = glob.sync(specsPath);

  if (specsGlob.length === 0) {
    logger.info('No spec files located. Stopping command from running.');
    return killServers(0);
  }

  ...
```